evap/staff/forms.py

```diff
@@ -506,6 +507,8 @@ def __init__(self, *args, evaluation=None, **kwargs):
             .distinct()
         )
 
+        self.can_be_deleted = self.instance.can_be_deleted and not self.instance.is_general
```

Comment:
Hmm I think it would be cool to have `can_be_deleted` be False on every general contribution. Could you maybe circumvent the issue with the new contributor by checking whether the instance has a primary key set? The new instance should not exits in the database, so it should not have a primary key set. For instances without a PK, I'd say it's fair that they can be deleted.

Reply:
Ok I see. I hope I unterstood it correctly. I made it so a general contribution can only be deleted if it has no pk of if a contribution is not general and has no answers which should be what we want (I hope) :)